src/odb/include/odb/db.h

```diff
@@ -7554,7 +7702,21 @@ class dbModule : public dbObject
 
   static dbModule* getModule(dbBlock* block_, uint dbid_);
 
-  std::string getHierarchicalName() const;
+  dbModInst* getModInst(dbId<dbModInst>);
+  dbInst* getdbInst(dbId<dbInst>);
```

Comment:
Why are these in terms of dbId?

Reply:
agreed ! There should be no dbId stuff in the user code, removed.